scripts/validate_docstrings.py

```diff
@@ -446,134 +450,157 @@ def validate_one(func_name):
     dict
         A dictionary containing all the information obtained from validating
         the docstring.
+
+    Notes
+    -----
+    The errors are codified in the next way:
+    - First two characters: Section where the error happens:
+       * GL: Global (no section, like section ordering errors)
+       * SS: Short summary
+       * ES: Extended summary
+       * PR: Parameters
+       * RT: Returns
+       * YD: Yields
+       * RS: Raises
+       * WN: Warns
+       * SA: See Also
+       * NT: Notes
+       * RF: References
+       * EX: Examples
+    - Last two characters: Numeric error code inside the section
+
+    For example, RT02 is the second codified error in the Returns section.
+
+    The exact codes are available in the source code of this function.
```

Comment:
If the "documentation" is the source code, can we have something like:

"For more information, please look at the source code of `parameter_mismatches` in `scripts/validate_docstrings.py`" (or something to that effect)

added somewhere in the error message so that new contributors won't be confused by the error output that they might see?

Reply:
I think that the contributors to the docstrings shouldn't care that much about the error codes. Hopefully the error messages are clear enough for them.

The reason for codifying the errors, is to be able to validate only specific errors in the CI (while we don't have all the docstrings in green for all errors).

We can add that message in the output of validating one docstring, but I don't even show the error codes there, so I think it would only add confusion.

Does it makes sense?

Reply:
Ah, right.  Good point.  Thanks for clarifying!